res/skins/LateNight/decks/rate_controls_compact.xml

```diff
@@ -113,8 +113,7 @@
                             <Size>5f,5f</Size>
                             <Pos>2,39</Pos>
                             <Connection>
-                              <ConfigKey><Variable name="Group"/>,rate</ConfigKey>
-                              <Transform><IsEqual>0.5</IsEqual></Transform>
+                              <ConfigKey><Variable name="Group"/>,rate_is_default</ConfigKey>
```

Comment:
`rate_set_default` or `rate_is_default`, what is it now?

Reply:
it's `rate_set_default`, https://github.com/ronso0/mixxx/pull/24 was incomplete.
fixed.